app/routes/crate/dependencies.js

```diff
@@ -6,7 +6,7 @@ export default class VersionRoute extends Route {
 
   async model() {
     let crate = this.modelFor('crate');
-    let versions = await crate.get('versions');
+    let versions = await crate.loadVersionsTask.perform();
 
     let { default_version } = crate;
     let version = versions.find(version => version.num === default_version) ?? versions.lastObject;
```

Comment:
Yeah, the part after #10288 needs to be changed. I'm also wondering if we could just load versions in the versions route. (This would have the downside of not having a versions count on the tab.)

Reply:
> I'm also wondering if we could just load versions in the versions route. (This would have the downside of not having a versions count on the tab.)

I forgot that we should the number of versions in the tab. I do look at that quite often though, so it would be unfortunate to lose it 🤔